packages/scene-system/sdk/NewSceneRuntime.ts

```diff
@@ -0,0 +1,165 @@
+import { LoadedModules, LoadableAPIs } from './../../shared/apis/client'
+import { initMessagesFinished, resolveMapping } from './Utils'
+import { LoadableParcelScene } from 'shared/types'
+import { customEval, getES5Context } from './sandbox'
+import { createFetch } from './Fetch'
+import { createWebSocket } from './WebSocket'
+import { RpcClient } from '@dcl/rpc/dist/types'
+import { PermissionItem } from 'shared/apis/gen/Permissions'
+import { sleep } from 'atomicHelpers/sleep'
+
+// New
+import { setupStats } from './new-rpc/Stats'
+import { createDecentralandInterface } from './new-rpc/DecentralandInterface'
+import { setupFpsThrottling } from './new-rpc/SetupFpsThrottling'
+import { createEventDispatcher, EventCallback, EventState } from './new-rpc/EventDispatcher'
+import { DevToolsAdapter } from './new-rpc/DevToolsAdapter'
+import type { EntityAction } from 'shared/apis/gen/EngineAPI'
+
+export async function startNewSceneRuntime(client: RpcClient) {
```

Comment:
There is a reason to use the word `New`?
The `New` is what it is here. And if we need to maintain old code like `decentraland-rpc`, I could agree to use `old-rpc`.

The same goes for the keyword `new-rpc`. Should be `rpc`, and the old rpc be renamed to `old-rpc`, and its code will be deleted one day.